### DISCUSSION TITLE
Branches view colours have gone, replaced by small dots

### DISCUSSION THREAD

**Discussion by Swizzle4372** [gitkraken/vscode-gitlens#3302] [Q&A] [answered]
Hi all.  I've been a long-time user of Gitlens and as long as I've been using it, branches shown in the Branches view have their entire line coloured based on syncronisity with the remote.  Something seems to have changed in the last few days and this is no longer happening for me.  Instead I'm just getting a small dot next to the branch icon. I also no longer see the coloured arrow icons.
Being colourblind, these are not easy to spot and I much preferred it when the entire line would change colour, ie go red when behind the remote.  Is there a way I can get this back?

Current View:
![image](https://github.com/gitkraken/vscode-gitlens/assets/19466487/50df1c10-13ab-4392-b915-bb4e54950858)

Example of how I want it to look:
![image](https://github.com/gitkraken/vscode-gitlens/assets/19466487/2fd661a4-e4c5-4856-b094-b5882e1b34b4)

Edited to add:  I'm on Gitlens v15.0.3
Thanks all!

**Comment by d13** [comment, marked as answer]
👋  Hi @ministe2003!

Thank you so much for posting! We got a fix in for that yesterday.

If you can, could you please try looking at branches view in today's pre-release edition of GitLens?

You can switch to the pre-release edition of GitLens, by clicking on the "Switch to Pre-Release version of this extension" from the Extensions view.

![228608740-6071bc19-7b1a-44c8-843f-d2af44d6114b](https://github.com/gitkraken/vscode-gitlens/assets/293282/386a9b38-e1b7-45a6-9204-3ae14995e5e1)

**Reply by Swizzle4372** [reply]
Thanks, only just getting round to this and I haven't needed to switch to pre-release since release is now 15.0.4 and the issue is fixed in this version.
Appreciate the quick reply!
